docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+The same applies if the reason for the previously failed database restore is corrected.
+When troubleshooting distributed database restore failures, it's important to collect logs from all database processes and their corresponding pods.
+
+> **NOTE**: In multi-cluster deployments, different backup cron jobs execute with different schedules unless they are configured to be part of the same backup group.
+
+Backup sets produced by different backup groups can't be used together to perform distributed database in-place restore.
+To perform a database restore in multi-cluster deployment, proceed with one of the approaches described below.
+
+1. [Select the hot copy SMs in one of the clusters](#fine-grained-archive-selection) and provide a backup set available in that cluster or `:group-latest` as `restore.source`.
+2. Make the selected backup set available as a remote URL in both clusters. It can be used as a restore source for all the SMs in the database only if no custom storage groups are configured.
+3. Configure HC SMs in all clusters to be part of a single backup group and leave one set of backup jobs enabled in only one of the clusters to control this backup group.
+This will ensure that all HC SMs backups will be coordinated during hot copy requests.
+
+### Fine-grained archive selection
+
+NuoDB _restore_ chart provides several ways to select which archives should be restored during [distributed database in-place restore](#distributed-database-in-place-restore) or [archive seed restore](#archive-seed-restore).
+
+- explicitly selecting specific archive IDs - `restore.archiveIds` variable can be set to specific archive IDs in the domain state for the target database.
+To list all archives for a database, the `nuocmd show archives --db-name <db>` is used.
+The process for each archive can be seen under the archive info which makes it easier for the user to mark the archives for restore.
+- selecting archives served by database processes with specific labels - `restore.labels` can be used to configure process labels, which then define the archives which will be selected for a restore.
+Any configured label and value that matches an SM process will add its archive to the list of selected archives for a restore.
+For instance, this can be used to easily select all SMs in a specific backup group - `--set restore.labels.backup="<backup group>"`.
+
+### Archive seed restore
+
+The archive seed restore operation restores a corrupted or lost archive while the database is running.
+This operation will reset the existing archive state, and once it joins the database, the new SM will sync to the current state of the running database.
+For this reason, you cannot restore the entire database by restoring a single SM in a running database.
+The _seed restore_ reduces the _SYNCing_ time from other running SMs as only the changed atoms will be transferred.
```

Comment:
in general, that's exactly what SYNC does too. In the absence of bugs, a full SM sync should be as fast as seed+diff-sync. I know that in reality, it is not.

But can we still change the wording to something like "The seed restore - has the potential to reduce - the syncing time"?

Reply:
I see the _archive seed restore_ only used when the archive needs to be re-initialized. It compares full archive sync with seed+diff sync here. In absence of bugs, the seed+diff sync should be a lot faster than transferring all atoms.
I agree that in any other use case it's hard to know what the gain will be so I will change the text as suggested.